### ISSUE TITLE
Memory leak in argtable3

### ISSUE THREAD

**Issue by plavin** [hpcgarage/spatter#70]
There is a memory leak in argtable3 that bothers the address sanitizer. To fix this, we can either (1) fork our own version since development seems to have ended on argtable anyways, or (2) try to get our fix pushed to their repo. 

Either way, it's probably not a real issue.

**Comment by jyoung3131** [comment]
@agustin-vaca is this still an issue with ASAN now that we've refactored how argtable is included in #117?

**Comment by agustin-vaca** [comment]
Getting similar errors with all tests when trying to run ASAN with OpenMP and Clang:
```
Error: Error: failed to allocate memory
Attempted to allocate 3000304 bytes (3000304 * 1)
```
on all Rogues Gallery machines I have access to.

**Comment by jyoung3131** [comment]
Closed due to imminent merging of refactor in #165 which removes the usage of argtable.
